projects/extension/Dockerfile

```diff
@@ -65,12 +65,6 @@ USER root
 
 RUN pip install --break-system-packages uv==0.5.20
 
-# install pgspot
-RUN set -eux; \
```

Comment:
@jgpruitt Is there any reason I'm not aware for building pgspot from source instead of installing as a dev dependency ?

Reply:
There was at least one time when pgspot had a bug that broke us. I installed it this way because I was able to go fix the bug, but I didn't want to wait on the publishing process.

Reply:
I guess it's fixed now 👍